druntime/src/importc.h

```diff
@@ -66,6 +66,18 @@
  */
 #define __extension__  /* ignore it, as ImportC doesn't do warnings */
 
+/********************************
+ * __has_extension is a clang thing:
+ *    https://clang.llvm.org/docs/LanguageExtensions.html
+ * ImportC doesn't has those extensions.
```

Comment:
```suggestion
 * ImportC doesn't have those extensions.
```

Reply:
Haha, I meant it the way I wrote it

Reply:
What's the meaning of intentionally wrong grammar here?

Reply:
A bad pun on `__has_extension`, dmd doesn't _has_ those extensions.

Reply:
Ohhh, of course